### PR TITLE
Css-pseudo-classes exercise - Heerak Moon

### PR DESCRIPTION
**Pull request by heerakmoon**
https://user-images.githubusercontent.com/113075380/214689403-404bc255-3f72-4662-8fe4-dad8336786eb.mp4

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+button {
+  font-size: x-large;
+  padding: 8px 25px;
+}
+
+button:hover {
```
Comment (robgmerrill) [inline css-pseudo-classes/styles.css:6]: On hover we want the cursor set to pointer.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Close!

**Comment by heerakmoon** [comment]
https://user-images.githubusercontent.com/113075380/214694545-c0f8c0e6-e2a5-4790-8fcf-3803d6d426a1.mp4

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**